docs/versioned_docs/version-2.50.0-LTS/widgets/tags.md

```diff
@@ -25,54 +25,54 @@ It can be used to set array of tags. It must be an array of objects like this:
 }}
 ```
 
-Each object should contain a title, color code of a particular tag, and also a text color.
+Each object should contain a different title in different background colors like success in green color, info in blue color, warning in orange color and danger in red color while a text color is white to all objects.
 
```

Comment:
The usage of the word `different` here makes it feel that you cannot have identical tags or identical titles or colours, which is not the actual behaviour.

Reply:
Hey @Vaishnavi-Raykar, please make the above change in both versions.
Thanks!

Reply:
@amanregu @RudraPratik30 Done Sir